tools/sigclient/pkg/query/query.go

```diff
@@ -541,6 +541,50 @@ func runContinuousQueries(client *http.Client, requestStr string, bearerToken st
 	}
 }
 
+func createString(value []interface{}) string {
+	var str string
+	for i, v := range value {
+		str += fmt.Sprintf("%v", v)
+		if i != len(value)-1 {
+			str += ","
+		}
+	}
+	return str
+}
```

Comment:
This is alright, but I think this would be better:
```
valueStrings := make([]string, 0, len(value))
for _, item := value {
    valueStrings = append(valueStrings, fmt.Sprintf("%v", v))
}

return strings.Join(valueStrings, ",")
```

Also, since you only call this function in one place, you might want to inline this code at that spot instead of making this function. You can have this separate function if you want though; in that case, I'd recommend renaming `createString` and `value` to something more clear

Reply:
Addressed in the latest commit.